microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/java/IJavaDiagnosticsParticipant.java

```diff
@@ -4,14 +4,15 @@
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v20.html
 *
+* SPDX-License-Identifier: EPL-2.0

```

Comment:
Do we need these headers for this project?

Reply:
When I move LSP4XML to Eclipse.org, this information missed. 

@fbricon we need to add this information, no?

Reply:
@xorye yes we need this header.